src/__test__/MissionList.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import MissionList from '../Components/MyMissions/MissionList';
+import '@testing-library/jest-dom';
+import store from '../Redux/configureStore';
+
+describe('Testing page deployment', () => {
+  test('Mission components matches snapshot', () => {
+    const tree = render(
+      <Provider store={store}>
+        <MissionList />
+      </Provider>,
+    );
+    expect(tree).toMatchSnapshot();
```

Comment:
- [ ] Good Job so far but almost there, I just noticed that some of the tests are not really passing and as required for this project you should write unit tests with React Testing Library and Jest (for unit tests). unfortunately, they can't all pass, I encourage you to take care of this for your project to match all the requirements for this project :wink:  this is the result below when trying to run the tests :point_down:  
![testfaild2](https://user-images.githubusercontent.com/88894525/218238318-6fc3f193-6aa8-4a4f-aea4-fc0db2761ae9.png)

![testfailed](https://user-images.githubusercontent.com/88894525/218238331-3b165c95-8403-4d22-acee-45700bb3756a.png)

Reply:
After correcting these are the results. Thanks for your comments!
![Captura de pantalla (313)](https://user-images.githubusercontent.com/79432268/218260085-baafb17f-af8d-4b5a-94a3-31fd442f71b7.png)